examples/java/veeam-vm-backup/README.MD

```diff
@@ -0,0 +1,144 @@
+
+
+## Prerequisites
+Install Veaam Backup & Replication and add your VC to the list of Veeam's managed servers.
+Details on how to do this can be found here: https://www.veeam.com/veeam_backup_10_0_quick_start_guide_vsphere_pg.pdf
+
+
+## Get the example function
+
+Clone this repository which contains the example functions. 
+
+```bash
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance
+cd vcenter-event-broker-appliance/examples/java/veeam-vm-backup
+git checkout master
+```
+
+## Customize the function
+
+For security reasons to not expose sensitive data we will create a Kubernetes secret (https://kubernetes.io/docs/concepts/configuration/secret/) which will hold Veeam credentials and information, as well as the vCenter ip . This secret will be mounted into the function during runtime. This is all taken care of for your by the appliance. We only have to create the secret with a simple command through `faas-cli`.
+
+First, change the configuration file `veeamconfig.toml` holding your secret vCenter information located in the veeam-vm-backup example folder:
+
+```toml
+# veeamconfig.toml contents
+# replace with your own values
+[enterprise_manager]
+ip = "VEEAM_ENTERPRISE_MANAGER_IP"
+port = "9398"	# This is the default port for Veeam's Restful API. Change it, if you changed the port during Veeam configuration. 
+user = "USERNAME"
+password = "PASSWORD"
+
+[backup_server]
+id = "BACKUP_SERVER_ID"  # Can be found through the request to your Veeam server described here: https://helpcenter.veeam.com/docs/backup/rest/get_backupservers.html?ver=100
+
+[vcenter]
+ip = "VC_IP"
+```
+
+Now go ahead and store this configuration file as secret in the appliance.
+
+```bash
+# set up faas-cli for first use
+export OPENFAAS_URL=https://VEBA_FQDN_OR_IP
+faas-cli login -p VEBA_OPENFAAS_PASSWORD --tls-no-verify # vCenter Event Broker Appliance is configured with authentication, pass in the password used during the vCenter Event Broker Appliance deployment process
+
+# now create the secret
+faas-cli secret create veeamconfig --from-file=veeamconfig.toml --tls-no-verify
+```
+
+**Note:** Delete the local `veeamconfig.toml` after you're done with this exercise to not expose this sensitive information.
+
+Lastly, define the function-specific settings. They are performed in the `veeam-vm-backup.yml` file. Open and edit the `veeam-vm-backup.yml` provided with in the Java veeam-vm-backup example code. Change `gateway` and (optionally) `topic` as per your environment/needs.
+
+```yaml
+version: 1.0
+provider:
+  name: openfaas
+  gateway: https://VEBA_FQDN_OR_IP # replace with your vCenter Event Broker Appliance environment
+functions:
+  veeam-vm-backup:
+    lang: java8
```

Comment:
Would the example also work with the `java11` template which is not marked deprecated and uses openjdk? https://github.com/openfaas/templates/tree/master/template/java11

Reply:
The function works with the java11 OpenFaaS template - details on this are added in the updated documentation.

Reply:
Can you please change the Java version for the image then to v11 as well please? v8 is marked deprecated and it would be great to have a function using an up to date image.

Reply:
After the change, please also update the README accordingly.

Reply:
Done.